src/applications/claims-status/tests/components/claim-files-tab/RemoveFileModal.unit.spec.jsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { expect } from 'chai';
+import sinon from 'sinon';
+import { $ } from '@department-of-veterans-affairs/platform-forms-system/ui';
+
+import RemoveFileModal from '../../../components/claim-files-tab/RemoveFileModal';
+
+describe('<RemoveFileModal>', () => {
+  it('should render component', () => {
+    const removeFileName = 'test.pdf';
```

Comment:
id make a props const that has all of the props for remove file modal that way you dont have to set them in each test ex:

const props = {
removeFileName: 'test.pdf',
showRemoveFileModal: true
}